sessionctx/binloginfo/binloginfo.go

```diff
@@ -73,6 +80,10 @@ func GetPrewriteValue(ctx sessionctx.Context, createIfNotExists bool) *binlog.Pr
 
 // WriteBinlog writes a binlog to Pump.
 func (info *BinlogInfo) WriteBinlog(clusterID uint64) error {
+	if atomic.LoadUint32(&errorStopped) != 0 {
+		return terror.ErrCritical.GenByArgs(errors.New("binilog already stopped"))
```

Comment:
If we return an error, how could we still provide service?

Reply:
The server will not hang now
https://github.com/pingcap/tidb/pull/6503/files#diff-dc417f5322100e2e83a8818c70a25878R481

Reply:
But it can not execute any write statement.